server/public/styles.css

```diff
@@ -2,10 +2,13 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-around;
-  margin-top: 200px;
   overflow: hidden;
 }
 
+.margin-top-page-spacing {
```

Comment:
This is the only weird thing in your code. You should be able to use a better name for this class. Try using class names that identify _what_ is being styled, not _how_. Maybe `.list-filters`?